### DISCUSSION TITLE
Source Code of Example Triangulated Surface Mesh Approximation

### DISCUSSION THREAD

**Discussion by HoBool** [CGAL/cgal#6957] [Q&A] [answered]
## Issue Details

![image](https://user-images.githubusercontent.com/62293522/195053788-81e61af0-330b-42cd-93c9-51bef2995313.png)

This example lookes very interesting, but i can't find the source code.
Do you still have the source code, maybe also the source mesh file ?

It would be very helpfull, if you can share it.

Thank you

**Comment by sloriot** [comment, marked as answer]
The doc of the package is [here](https://doc.cgal.org/latest/Surface_mesh_approximation/index.html#Chapter_Triangulated_Surface_Mesh_Approximation).

Sample code is provided [here](https://doc.cgal.org/latest/Surface_mesh_approximation/index.html#title13)
